docs/.templates/integration/schema.json

```diff
@@ -0,0 +1,89 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Netdata Integrations file",
+    "properties": {
+        "categories": {
+            "type": "object",
+            "description": "A list defining all the available categories for the integrations.",
+            "properties": {
+                "list": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "description": "The concept of IDs between categories is used, so we can handle nested categories. That way they will be able to mention their parent, and we should be able to make the structure of categories. Every category's ID will be used by the various integrations too to mention in which category they lie. Along with this, a priority field is needed to define the ordering on a menu-level.",
+                        "properties": {
+                            "name": {
+                                "type": "string",
+                                "description": "Name of the category. This is the actual name of the category or subcategory, (e.g. Web Servers, Operating Systems, Databases)."
+                            },
+                            "priority": {
+                                "type": "string",
+                                "description": "Priority of the category. A number expressing where the category should be in the menu. The smaller the number the higher the priority, so a category with priority 1 would be above a category with a priority of 50 in terms of menu ordering."
+                            },
+                            "id": {
+                                "type": "string",
+                                "description": "ID of the category that will be used by integrations, this is a unique number for each category that will be used by its children to refer to it as their parent."
+                            },
+                            "parent_id": {
+                                "type": [
+                                    "null",
+                                    "string"
+                                ],
+                                "description": "The category's parent ID. If this category is a subcategory, then it needs to mention its parent, where it belongs to. Make it null if no parent and this is a top level category."
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "integrations": {
+            "type": "array",
+            "description": "A list of integration elements, combination of metadata.yaml and other sources.",
+            "items": {
+                "type": "object",
+                "properties": {
```

Comment:
Maybe we should add a `priority<Integer>` field here in order to be able to control the order the integrations will show up in each category